src/app/services/keybinding.service.ts

```diff
@@ -0,0 +1,156 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+import { Injectable } from '@angular/core';
+
+import { Subject, Observable } from 'rxjs/Rx';
```

Comment:
```typescript
import { Subject, Observable } from 'rxjs/Rx';
```
 works because of `rxjs-compat`.
In new code we should use 
```typescript
import { Subject, Observable } from 'rxjs';
```

Reply:
When I do that, I get a

externals.js:389 ERROR TypeError: s.Observable.fromEvent is not a function

Reply:
Solved